coral/src/app/layout/SidebarNavigation.test.tsx

```diff
@@ -0,0 +1,63 @@
+import SidebarNavigation from "src/app/layout/SidebarNavigation";
+import { cleanup, screen, render, within } from "@testing-library/react";
+
+// mock out svgs to avoid clutter
+jest.mock("@aivenio/design-system", () => {
+  return {
+    __esModule: true,
+    ...jest.requireActual("@aivenio/design-system"),
+
+    Icon: () => {
+      return <div data-testid={"ds-icon"}></div>;
+    },
+  };
+});
+
+// "http://localhost/" comes from window.location.origin
+// that represents the Angular app
+const navLinks = [
+  {
+    name: "Overview",
+    linkTo: "http://localhost/index",
+  },
+  { name: "Topics", linkTo: "/" },
+  { name: "Kafka Connector", linkTo: "http://localhost/kafkaConnectors" },
+  { name: "Schemas", linkTo: "/" },
+  { name: "User and teams", linkTo: "/" },
+  { name: "Audit log", linkTo: "/" },
+  { name: "Settings", linkTo: "/" },
+];
+
+describe("SidebarNavigation.tsx", () => {
+  beforeAll(() => {
+    render(<SidebarNavigation />);
+  });
+
+  afterAll(cleanup);
+
+  it("renders the primary navigation", () => {
+    const nav = screen.getByRole("navigation", { name: "Primary navigation" });
```

Comment:
`Primary navigation` -> `Main navigation`. What do you think? Splitting hairs here, but I think _main_ less engineery 🤔